source/introduction.md

```diff
@@ -0,0 +1,16 @@
+# Introduction
```

Comment:
Sphinx supports reStructuredText by default, and I added plugin to support Markdown (CommonMark) because I thought that many GitHub user has got used to play with `.md` files.

In many cases `.md` should be enough, but `.rst` has colourful features including tables so sometimes we may need it.

I have no experience to use Asciidoc with Sphinx for now. Official document provides [no help](http://www.sphinx-doc.org/en/1.5/search.html?q=asciidoc&check_keywords=yes&area=default), and SO also provides nothing. It seems that we cannot use asciidoc with Sphinx...